apps/full-site-editing/full-site-editing-plugin/wp-template.php

```diff
@@ -60,6 +61,98 @@ function fse_register_wp_template() {
+				'not_found_in_trash'       => __( 'No template parts found in Trash.' ),
+				'filter_items_list'        => __( 'Filter template parts list' ),
+				'items_list_navigation'    => __( 'Template parts list navigation' ),
+				'items_list'               => __( 'Template parts list' ),
+				'item_published'           => __( 'Template part published.' ),
+				'item_published_privately' => __( 'Template part published privately.' ),
+				'item_reverted_to_draft'   => __( 'Template part reverted to draft.' ),
+				'item_scheduled'           => __( 'Template part scheduled.' ),
+				'item_updated'             => __( 'Template part updated.' ),
+			),
+			'menu_icon'             => 'dashicons-layout',
+			'public'                => false,
+			'show_ui'               => true,
+			'show_in_menu'          => true,
+			'rewrite'               => false,
+			'show_in_rest'          => true,
+			'rest_base'             => 'template_parts',
+			'rest_controller_class' => 'A8C_REST_Templates_Controller',
+			'capability_type'       => 'template_part',
+			'capabilities'          => array(
+				// You need to be able to edit posts, in order to read templates in their raw form.
+				'read'                   => 'edit_posts',
+				// You need to be able to customize, in order to create templates.
+				'create_posts'           => 'edit_theme_options',
+				'edit_posts'             => 'edit_theme_options',
+				'delete_posts'           => 'edit_theme_options',
+				'edit_published_posts'   => 'edit_theme_options',
+				'delete_published_posts' => 'edit_theme_options',
+				'edit_others_posts'      => 'edit_theme_options',
+				'delete_others_posts'    => 'edit_theme_options',
+				'publish_posts'          => 'edit_theme_options'
+			),
+			'map_meta_cap'          => true,
+			'supports'              => array(
+				'title',
+				'editor',
+			),
+		)
+	);
+
+	register_taxonomy(
+		'wp_template_part_type',
+		'wp_template_part',
+		array(
+			'labels'            => array(
+				'name'              => _x( 'Template Part Types', 'taxonomy general name' ),
+				'singular_name'     => _x( 'Template Part Type', 'taxonomy singular name' ),
+				'menu_name'         => _x( 'Template Part Types', 'admin menu' ),
+				'all_items'         => __( 'All Template Part Types' ),
+				'edit_item'         => __( 'Edit Template Part Type' ),
+				'view_item'         => __( 'View Template Part Type' ),
+				'update_item'       => __( 'Update Template Part Type' ),
+				'add_new_item'      => __( 'Add New Template Part Type' ),
+				'new_item_name'     => __( 'New Template Part Type' ),
+				'parent_item'       => __( 'Parent Template Part Type' ),
+				'parent_item_colon' => __( 'Parent Template Part Type:' ),
+				'search_items'      => __( 'Search Template Part Types' ),
+				'not_found'         => __( 'No template part types found.' ),
+				'back_to_items'     => __( 'Back to template part types' ),
+			),
+			'public'            => true,
```

Comment:
What do you think about keeping it private? 🤔 It seems to me like an implementation detail that we don't necessarily want to expose to front-end users.